pandas/tests/indexes/multi/test_contains.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 import pytest
+import re
```

Comment:
you need to isort imports. code check validation is failing ci.

Reply:
I saw that, but did not understand what it's looking for. Alphabetically `numpy, pytest, re` is correct, yes?

Reply:
Anyway, I made a guess that it was failing because the check was put in after the last time the file changed, so I tried alphabetically sorting the remaining `import`s. If that still needs fixing, please point me towards instructions on how to do so

Reply:
Annnd now I see that `isort` is in fact a whole library for nothing other than sorting imports. That's kind of horrifying, though I did end up writing a short script to do just that the other day. I'll see what its output has to say

Reply:
This seems to be resolved. I ran `isort` on the offending file, and now the lint check is passing.